lib/util/text_style_ext.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+
+extension TextStyleExtension on TextStyle {
+  TextStyle adjustSize(double size) {
+    return copyWith(fontSize: fontSize! + size);
```

Comment:
I think you might need to check the `min` size here. See some small font sizes can become  very small or negative values.